package.json

```diff
@@ -42,7 +42,7 @@
     "watch": "babel --optional runtime resources/watch.js | node",
     "pretty": "node resources/pretty.js",
     "pretty-check": "node resources/pretty.js --check",
-    "preversion": ". ./resources/checkgit.sh && npm test",
+    "preversion": ". ./resources/checkgit.sh && yarn test",
```

Comment:
@benjie because this project has a `yarn.lock` file, no one should be doing development without `yarn`

Reply:
I don't agree; with libraries the lockfile can actually be misleading because the lockfile is not distributed, so dependents may install different software versions than then library developers. If we can get bad package resolutions just using package.json then we may have problems we need to solve (devDependencies notwithstanding).

If someone is not changing the dependencies (which we require they do via `yarn`), there should be no reason that they can't contribute without installing yarn. I'd still encourage them to use yarn, but we should avoid it being a requirement unless there's a really good reason, IMO.

That said, this opinion is weakly held, so feel free to overrule.